src/Chapter2.hs

```diff
@@ -620,7 +636,12 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven [] = []
+takeEven [x] = [x]
+takeEven [x, _] = [x]
```

Comment:
`[x, _]` is syntax sugar for `(x : _ : [])` and you can see that this is already covered by your last line. It's a good idea to cover some small cases manually to see the general pattern first :slightly_smiling_face: But in the end, you can remove all redundant cases